subprojects/docs/src/docs/userguide/troubleshooting/validation_problems.adoc

```diff
@@ -0,0 +1,12 @@
+[[validation-problem]]
```

Comment:
I suppose we need to include this page from somewhere, so it is part of our documentation? Or how does this work right now?

Reply:
It's in the generated docs but currently it would be unllisted. So basically only those seeing the error messages would get a link.